api/v1/storagecluster_types.go

```diff
@@ -254,6 +254,7 @@ type ManageCephRBDMirror struct {
 type MgrSpec struct {
 	// EnableActivePassive can be set as true to deploy 2 ceph manager pods, one active and one standby
 	// Ceph will promote the standby mgr when the active mgr goes down due to any reason
+	// +kubebuilder:deprecatedversion:warning="This field has been deprecated and will be removed in future. By default we now have 2 ceph manager pods, one active and one standby ."
```

Comment:
There is no way for the user to change the number of mgrs, right? So it seems better not to mention "by default" since that implies they could change it.

Reply:
Although with the basic profile we will allow a single mgr, so this comment would still be valid with the default profile.